src/DGtal/geometry/surfaces/estimation/VoronoiCovarianceMeasureOnDigitalSurface.ih

```diff
@@ -127,7 +127,7 @@ VoronoiCovarianceMeasureOnDigitalSurface( ConstAlias< Surface > _surface,
         {
           Point p = *itPts;
           const EigenStructure& evcm = myPt2EigenStructure[ p ];
-          VectorN n = evcm.vectors.column( 2 );
+          VectorN n = evcm.vectors.column( Space::dimension-1 );
```

Comment:
@JacquesOlivierLachaud I'm not sure to understand what is going on here ... can you confirm that it is what you meant ?